src/main/java/core/basesyntax/GeometricFigure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class GeometricFigure implements Drawable, AreaCalculator {
```

Comment:
Please pay attention to the task requirements which ask you to create a Figure class, not GeometricFigure